client_plugin/drivers/shared/shared_driver.go

```diff
@@ -124,10 +124,8 @@ func NewVolumeDriver(cfg config.Config, mountDir string) *VolumeDriver {
 	}
 
 	// Load the file server image
-	if d.dockerOps.LoadFileServerImage() == false {
-		return nil
-	}
-	log.Infof("Loaded file server image")
+	go d.dockerOps.LoadFileServerImage()
```

Comment:
I assume "initialize built-in etcd cluster" is not dependent on whether loading file image is done or not, right?